### PR TITLE
Honor modifiers in multitap detection

### PR DESCRIPTION
**Pull request by ubitux**
Fixes Issue #15577.

### CODE REVIEW

File: OpenRA.Platforms.Default/MultiTapDetection.cs [hunk 1]
```diff
@@ -31,12 +31,12 @@ public static int InfoFromMouse(byte button)
 			return clickHistoryCache[button].LastTapCount();
 		}
 
-		public static int DetectFromKeyboard(Keycode key)
+		public static int DetectFromKeyboard(Pair<Keycode, Modifiers> key)
```
Comment (pchote) [inline OpenRA.Platforms.Default/MultiTapDetection.cs]: IMO it would be cleaner if this took in two arguments rather than a `Pair`.

**Review by pchote — COMMENTED** [review]
Looks reasonable to me, just one minor style nit to suggest:

File: OpenRA.Platforms.Default/MultiTapDetection.cs (hunk 1 above)
Comment (ubitux) [reply OpenRA.Platforms.Default/MultiTapDetection.cs]: Sure, fixed

**Review by pchote — APPROVED** [review]

**Review by obrakmann — APPROVED** [review]

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/db96f2d81191831b9da438dc2b16972ade7c3f76)

**Pull request merged**